cluster/src/test/scala/io/snappydata/benchmark/snappy/tpch/QueryExecutor.scala

```diff
@@ -125,28 +122,32 @@ object QueryExecutor {
             else s"${threadNumber}_QueryPlans_Spark.out"
     val queryResultsFileName = if (isSnappy) s"${threadNumber}_Snappy_Q${queryNumber}_Results.out"
             else s"${threadNumber}_Spark_Q${queryNumber}_Results.out"
-    val queryStatisticsFileName = if (isSnappy) s"${threadNumber}_Snappy_Q${queryNumber}_Timings.csv"
+    val queryStatisticsFileName =
+      if (isSnappy) s"${threadNumber}_Snappy_Q${queryNumber}_Timings.csv"
             else s"${threadNumber}_Spark_Q${queryNumber}_Timings.csv"
 
     if (planFileStream == null && planPrintStream == null) {
       planFileStream = new FileOutputStream(new File(planFileName))
       planPrintStream = new PrintStream(planFileStream)
     }
 
-    val queryStatisticsFileStream: FileOutputStream = new FileOutputStream(new File(queryStatisticsFileName))
+    val queryStatisticsFileStream: FileOutputStream =
+      new FileOutputStream(new File(queryStatisticsFileName))
     val queryStatisticsPrintStream: PrintStream = new PrintStream(queryStatisticsFileStream)
     queryStatisticsPrintStream.println(s"Iteration,ResponseTime")
     // scalastyle:off println
     try {
       println(s"Started executing $queryNumber")
```

Comment:
Statement "Started executing query: $queryNumber" would be more clear.

Reply:
Okay.